packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -105,6 +106,7 @@ export const useScaffoldEventHistory = <
         eventName,
         fromBlock: fromBlock.toString(),
         chainId: targetNetwork.id,
+        filters: JSON.stringify(filters, replacer),
```

Comment:
We could have passed the `filters` obj directly but sometimes there can be bigInt init it too. When react-query make a queryHash of all this items it uses JSON.stringify internally and not able to parse bigInt. 

So here we are making the react-query's life easier by handling the object with bigInt earlier so its easier for it to create query hash